shell/list/projectsecret.vue

```diff
@@ -132,10 +132,8 @@ export default {
      */
     filterRowsLocal(rows: Secret[]) {
       return rows.filter((r: Secret) => {
```

Comment:
This wasn't functioning at all on my end. I'm sure this needs to be different though.

Reply:
note there's some backend issues that make the result set a bit weird, still waiting on those to verify vai on world